vscode-extensions/project-explorer/extension/package.json

```diff
@@ -0,0 +1,55 @@
+{
+  "name": "vscode-project-explorer-extension",
+  "private": true,
+  "version": "11.2.0",
+  "description": "VSCode extension that provides the Axon Ivy project explorer",
+  "displayName": "Axon Ivy Project Explorer",
+  "publisher": "AxonIvy",
```

Comment:
Axon Ivy AG

Reply:
not possible:
![image](https://github.com/axonivy/vscode-extensions/assets/93579455/4b3fe22a-536c-42d9-a98d-7be49a8f3bfa)

Reply:
what are the restrictions?

Reply:
Expected the identifier of a publisher, not its human-friendly name.  Learn more: https://code.visualstudio.com/api/working-with-extensions/publishing-extension#publishing-extensions